packages/p2p/src/components/my-ads/edit-ad-form.jsx

```diff
@@ -58,11 +62,25 @@ const EditAdForm = () => {
               selected_methods.length === Object.keys(payment_method_details).length
           );
 
+    const handleCancelEdit = is_form_edited => {
+        if (is_form_edited || payment_methods_changed) {
+            setIsCancelEditModalOpen(true);
+        } else {
+            my_ads_store.setShowEditAdForm(false);
+        }
+    };
+
+    const confirmCancelEdit = is_cancel_edit =>
+        is_cancel_edit ? my_ads_store.setShowEditAdForm(false) : setIsCancelEditModalOpen(false);
+
```

Comment:
We're passing `confirmCancelEdit` to the prop `handleCancelEdit` in another file while this `handleCancelEdit` is being used in Formik? Can you update these naming conventions.